### PR TITLE
Review "Team LBAC" page

### LINKED ISSUE #542: Review `docs/sources/administration/data-source-management/teamlbac/_index.md`
https://github.com/grafana/grafana/blob/main/docs/sources/administration/data-source-management/teamlbac/_index.md

### PR DESCRIPTION
**Pull request by jdbaldry**
- Use semantic line breaks for better line-based diffing in GitHub in the future.
- Use consistent and ordered heading levels.
- Avoid first person.
- Generally avoid passive voice.
- Remove FAQ section as all the questions are answered elsewhere in the page.

Closes https://github.com/grafana/writers-toolkit/issues/542

Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

File: docs/sources/administration/data-source-management/teamlbac/_index.md [hunk 1]
```diff
@@ -14,71 +14,55 @@ weight: 100
...
+1. Having a high number of Grafana Cloud data sources.
+   Team LBAC lets Grafana administrators reduce the total number of data sources per instance from hundreds, to one.
+1. Using the same dashboard across multiple teams.
+   Team LBAC lets Grafana Teams use the same dashboard with different access control rules.
 
-**Team LBAC** in the context of Loki, is a way to control access to logs based on labels present depending on the rules set for each team. Users wanting fine grained access to their logs in Loki, can now configure their users access based on their team memberships via **LogQL**.
+To set up Team LBAC for a Loki data source, refer to [Configure Team LBAC](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/data-source-management/teamlbac/configure-teamlbac-for-loki/).
 
-This feature addresses two common challenge faced by Grafana users:
+{{< admonition type="note" >}}
```
Comment (jdbaldry) [inline docs/sources/administration/data-source-management/teamlbac/_index.md:31]: Reviewing this patch, I think that I should probably put this back at the top of the page

**Review by eleijonmarck — APPROVED** [review]
i'd emphasize the FAQ as a way to see easily the answers for some of our users questions.

But this can be worked on later as well. if this is not applicable of a section 👍 

Thx for the review of the docs.

Everything seems to be in order and I checked the feature details, and nothing overall has changed to removing some feature set of Team LBAC

**Comment by jdbaldry** [comment]
Thank you for taking a look. I'm actually not certain on whether I should have removed the FAQ because I agree it prominently answered some important reader questions. The only reference I have is https://github.com/grafana/writers-toolkit/issues/293 at the moment but it seems to be more focused on a separate FAQs page for a project.

**Comment by eleijonmarck** [comment]
@jdbaldry this should go out w. 10.4 btw

**Comment by jdbaldry** [comment]
I'm gonna drop the CODEOWNERs changes so that this PR doesn't get stalled.

**Pull request merged**